test/runtests.jl

```diff
@@ -1,32 +1,67 @@
-using NLPModels, OptimizationProblems, Test
+using NLPModels, NLPModelsJuMP, OptimizationProblems, Test
+
+function vio(c, l, u)
+  return max.(max.(c - u, 0), max.(-c + l, 0))
+end
+
+@test names(OptimizationProblems.ADNLPProblems) == [:ADNLPProblems]
+
+import ADNLPModels
+
+ndef = OptimizationProblems.ADNLPProblems.default_nvar
 
 # Test that every problem can be instantiated.
 for prob in names(OptimizationProblems.PureJuMP)
   prob == :PureJuMP && continue
   println(prob)
   prob_fn = eval(Meta.parse("OptimizationProblems.PureJuMP.$(prob)"))
-  prob_fn()
-end
+  model = prob_fn(ndef)
 
-@test OptimizationProblems.ADNLPProblems.problems == []
+  prob == :fminsrf2 && continue # issue because variable is a matrix!
+  prob == :hs61 && continue #because nlpmodelsjump is not working here https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl/issues/84
+  prob in [:clplatea, :clplateb, :clplatec] && continue # issue because variable is a matrix!
 
-import ADNLPModels
+  nlp_jump = MathOptNLPModel(model)
+  nlp_ad = eval(Meta.parse("OptimizationProblems.ADNLPProblems.$(prob)()"))
 
-ndef = OptimizationProblems.ADNLPProblems.default_nvar
-for prob in OptimizationProblems.ADNLPProblems.problems
+  @test (nlp_jump.meta.nvar == nlp_ad.meta.nvar) && (nlp_jump.meta.x0 == nlp_ad.meta.x0)
+  @test nlp_jump.meta.ncon == nlp_ad.meta.ncon
+  @test (nlp_jump.meta.lvar == nlp_ad.meta.lvar) && (nlp_jump.meta.uvar == nlp_ad.meta.uvar)
```

Comment:
```suggestion
  @test nlp_jump.meta.nvar == nlp_ad.meta.nvar
  @test nlp_jump.meta.x0 == nlp_ad.meta.x0
  @test nlp_jump.meta.ncon == nlp_ad.meta.ncon
  @test nlp_jump.meta.lvar == nlp_ad.meta.lvar
  @test nlp_jump.meta.uvar == nlp_ad.meta.uvar
```
It's easier to find out where's the issue if it fails.

Reply:
Thanks!